CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+1.3.1 Release notes (2021-01-04)
+=============================================================
+
+### Fixed
+
+* Fix removing parent from view when system create new RingPublishingGDPRActivity before destroy old one.
```

Comment:
Fix for removing parent from view when system created new RingPublishingGDPRActivity before destroying old one.

Reply:
Zmienione